src/Views/batchprocessor.html

```diff
@@ -279,6 +279,7 @@ <h3>Select Basin Characteristics</h3>
                                 <th style="text-align: center;"><small>Time Submitted (UTC)</small></th>
                                 <th style="text-align: center;"><small>Time Started (UTC)</small></th>
                                 <th style="text-align: center;"><small>Time Completed (UTC)</small></th>
+                                <th style="text-align: center;"><small>Upload File Name</small></th>

```

Comment:
I think I'd just call it File Name

```suggestion
                                <th style="text-align: center;"><small>File Name</small></th>

```